api/net/Program.cs

```diff
@@ -67,7 +67,7 @@
       options.JsonSerializerOptions.PropertyNamingPolicy = jsonSerializerOptions.PropertyNamingPolicy;
       options.JsonSerializerOptions.WriteIndented = jsonSerializerOptions.WriteIndented;
       options.JsonSerializerOptions.Converters.Add(new JsonStringEnumConverter());
-      options.JsonSerializerOptions.Converters.Add(new Int32ToStringJsonConverter());
+      //   options.JsonSerializerOptions.Converters.Add(new Int32ToStringJsonConverter());
```

Comment:
Temporarily testing removal to see if in Openshift this resolves the OOM issue.  I don't know if we need this serializer.  I hope we don't, but it could be needed for some of the services.